src/components/ConversationSettings/LinkShareSettings.vue

```diff
@@ -103,9 +119,11 @@ export default {
 		NcButton,
 		NcCheckboxRadioSwitch,
 		NcPasswordField,
-		ArrowRight,
+		// Icons
```

Comment:
Rename remaining two icons and sort alphabetically before Nc.. components?

Reply:
We usually put icons after components